chaste_codegen/tests/chaste_test_utils.py

```diff
@@ -21,27 +22,19 @@ def load_chaste_models(model_types=[], ref_path_prefix=['chaste_reference_models
             if model_file.endswith('.cellml'):  # make sure we only process .cellml files
                 model_name_from_file = model_file.replace('.cellml', '')
                 model_file = os.path.join(model_folder, model_file)
-                reference_models = {}
                 for model_type in model_types:
-                    expected_hpp_path = os.path.join(cg.DATA_DIR, 'tests')
-                    expected_cpp_path = os.path.join(cg.DATA_DIR, 'tests')
-                    for pref in ref_path_prefix:
-                        expected_hpp_path = os.path.join(expected_hpp_path, pref)
-                        expected_cpp_path = os.path.join(expected_cpp_path, pref)
+                    expected_hpp_path = os.path.join(cg.DATA_DIR, 'tests', reference_folder)
+                    expected_cpp_path = os.path.join(cg.DATA_DIR, 'tests', reference_folder)
                     expected_hpp_path = os.path.join(expected_hpp_path, model_type, model_name_from_file + '.hpp')
                     expected_cpp_path = os.path.join(expected_cpp_path, model_type, model_name_from_file + '.cpp')
 
                     # Skip cellml files without reference chaste code
                     if os.path.isfile(expected_hpp_path) and os.path.isfile(expected_cpp_path):
-                        reference_models.update({model_type: {'expected_hpp_path': expected_hpp_path,
-                                                'expected_cpp_path': expected_cpp_path}})
-
-                if len(reference_models) > 0:
-                    class_name = class_name_prefix + model_name_from_file
-                    model_files.append({'model': cellmlmanip.load_model(model_file),
-                                        'model_name_from_file': model_name_from_file,
-                                        'class_name': class_name + 'FromCellML',
-                                        'reference_models': reference_models})
+                        model_files.append({'model_type': model_type,

```

Comment:
Indenting issue? (not 4 spaces)

Reply:
you're looking at indentation of deleted code